src/Range.js

```diff
@@ -1,31 +1,26 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 
-class Range extends React.Component {
-  componentDidMount() {
-    if (typeof M !== 'undefined') {
-      this.instance = M.Range.init(this._range);
-    }
-  }
+const Range = props => {
+  const instance = useRef(null);
+  const rangeRef = useRef(null);
```

Comment:
instead of `this.instance` we use `useRef`, more info: https://reactjs.org/docs/hooks-faq.html#is-there-something-like-instance-variables